k3s version v1.21.4+k3s1 (3e250fdb) go version go1.16.6 does not crea… (#3514)
rancher/docs: content/k3s/latest/en/backup-restore/_index.md

Snapshots are enabled by default.

> The snapshot directory defaults to `${data-dir}/server/db/snapshots`. Note that the `/var/lib/rancher/k3s` prefix can be changed by setting the `--data-dir` flag.

Reviewer: ```suggestion
The snapshot directory defaults to `${data-dir}/server/db/snapshots`. The data-dir value defaults to `/var/lib/rancher/k3s`, and can be changed by setting the `--data-dir` flag.
```

Reviewer 2: Thanks as always for the feedback!

> The snapshot directory defaults to `${data-dir}/server/db/snapshots`. Note that the `/var/lib/rancher/k3s` prefix can be changed by setting the `--data-dir` flag. This prefix is currently omitted from the documented paths.

Reviewer: If this PR adds it to both the places it was missing from, we can probably drop that bit.

Reviewer 2: Removed phrase, thanks!
